jailer/src/env.rs

```diff
@@ -301,11 +302,12 @@ impl Env {
 
         Err(Error::Exec(
             Command::new(chroot_exec_file)
-                .arg(format!("--id={}", self.id))
-                .arg(format!("--seccomp-level={}", self.seccomp_level))
-                .arg(format!("--start-time-us={}", self.start_time_us))
-                .arg(format!("--start-time-cpu-us={}", self.start_time_cpu_us))
-                .arg(format!("--api-sock=/{}", socket_file_name))
+                .args(&["--id", &self.id])
+                .args(&["--seccomp-level", &self.seccomp_level.to_string()])
+                .args(&["--start-time-us", &self.start_time_us.to_string()])
+                .args(&["--start-time-cpu-us", &self.start_time_cpu_us.to_string()])
+                .arg("--api-sock")
+                .arg(format!("/{}", socket_file_name))
```

Comment:
nit: are these changes needed ?. Personally I find the previous version more readable.

Reply:
Unfortunately yes, because `clap` accepts also the `--arg=value` option for passing parameters, our implementation does not.